src/android/io/branch/BranchSDK.java

```diff
@@ -167,11 +167,7 @@ public boolean execute(String action, JSONArray args, CallbackContext callbackCo
                     cordova.getThreadPool().execute(r);
                     return true;
                 } else if (action.equals("showShareSheet")) {
-                    if (args.length() == 4) {
-                        this.showShareSheet(args.getInt(0), args.getJSONObject(1), args.getJSONObject(2), args.getString(3));
-
-                        return true;
-                    } else {
+                    if (args.length() < 3) {
```

Comment:
Shouldn't this be length < 4? It would never go on the error branch if it is called with 3 arguments.